src/twisted/python/test/test_sendmsg.py

```diff
@@ -397,7 +397,7 @@ def test_shortsend(self):
         """
         L{send1msg} returns the number of bytes which it was able to send.
         """
-        message = "x" * 1024 * 1024
+        message = "x" * 1024 * 1024 * 8
```

Comment:
Please update the comment below to say "8MB" (or to not mention the size and to refer to the code).